### DISCUSSION TITLE
Permissions?

### DISCUSSION THREAD

**Discussion by naveenbharadwaj19** [CariusLars/ar_flutter_plugin#43] [Q&A] [answered]
What are the permission need to be added in manifest and gradle ?

**Comment by CariusLars** [comment, marked as answer]
Once you import this plugin into your Flutter app there should be no need to further add any permissions. If you use the cloud anchor feature,  required permissions are described in the [CloudAnchorSetup](https://github.com/CariusLars/ar_flutter_plugin/blob/main/cloudAnchorSetup.md) document, a fix for a possible permission error on iOS devices is described in the [Readme](https://github.com/CariusLars/ar_flutter_plugin/blob/main/README.md), but generally, it should work out of the box. Let me know if there are any problems!
